core/src/main/java/org/pentaho/platform/dataaccess/datasource/api/resources/JDBCDatasourceResource.java

```diff
@@ -52,7 +50,7 @@
 @Path( "/data-access/api/datasource/jdbc/connection" )
 public class JDBCDatasourceResource {
 
-  protected ConnectionServiceImpl service;
+  private ConnectionServiceImpl service;
```

Comment:
Changing back to private. This was changed from private to protected for unit tests only.
Not necessary, see new unit tests.